src/winforms/toga_winforms/keys.py

```diff
@@ -18,33 +19,16 @@ def toga_to_winforms_key(key):
     Key.MOD_1: WinForms.Keys.Control,
     Key.MOD_2: WinForms.Keys.Alt,
 }
+WINFORMS_MODIFIERS_MAP.update({
```

Comment:
The name "modifiers" is a little misleading here; keys like Up and Down aren't really modifiers. If I'm reading this right, this is really a map of "non-printable" characters, or something like that. You may also be able to use Toga.Key.is_printable(), or a single iteration pass through Toga.Key that handles printable and unprintable characters differently